deblur/test/test_workflow.py

```diff
@@ -576,63 +568,65 @@ def test_build_index_sortmerna(self):
-                (10, 0): 7, (11, 0): 9}
-        otu_ids = ["TACCTGCAGCCCAAGTGGTGGTCGATTTTATTGAGTCTAAAACGTTCGTAGCCGG"
-                   "TTTGATAAATCCTTGGGTAAATCGGGAAGCTTAACTTTCCGATTC",
-                   "CACCGGCGGCCCGAGTGGTGGCCATTATTATTGGGTCTAAAGGGTCCGTAGCCGG"
-                   "TTTGATCAGTCTTCCGGGAAATCTGACAGCTCAACTGTTAGGTCT",
-                   "CACCAGCAGCCCAAGTGGTGGCCGGGTTTATTGGGCCTAAAGCGCTCGTAGCCGG"
-                   "ACTGGTAAGTCCCTTGTGAAATCGGTCGGCTCAACCGTTCGGTGC",
-                   "TACCAGCCCCGCGAGTGGTCAGGACGATTATTAAGCCTAAGGCGTCCGTAGCCGG"
-                   "CCTTGTAAGTTCTCCCTTAAATGGTATGGCTCAACCATACCGTGG",
-                   "TTCCGGTCCCAAGCCGAACAGTTTCCGCCGGACGCCCATCCGTTGAGCGGATGGA"
-                   "TTTCCCGACAGACTTGTTCGGCCAGCTACGGACGCTTTAGGCCCA",
-                   "GTGCCAGCCGCCGCGGTAATACCCGCAGCTCAAGTGGTGGTCGCTATTATTGAGC"
-                   "CTAAAACGTCCGTAGTCGGCTTTGTAAATCCCTGGGTAAATCGGG",
-                   "TACCAGCCCCTTAAGTGGTAGGGACGATTATTTGGCCTAAAGCGTCCGTAGCCGG"
-                   "CTTAGTGAGTCTTTCGTTAAATCCGGCGACTTAATCGTCGTTTGC",
-                   "CGCCCGGTCCCAAGCTTTGCAGTCTCCCCAGAAATCGGATCGTTAAGCGACCCGA"
-                   "TTTACCCAGGGATTTACAAAGCCGACTACGGACGTTTTAGGCTCA",
-                   "TACCGGCAGCTCGAGTGATGACCGCTATTATTGGGCCTAAAGCGTCCGTAGCTGG"
-                   "CCGAACAAGTCTGTCGGGAAATCCATCCGCTCAACGGATGGGTCC",
-                   "TACCAGCTCTCCGAGTGGTGGGGACGATTATTGGGCTTAAAGCGTTCGTAGCCGG"
-                   "CTCAACAAGTCTCCCGTTAAATCCAGCGATCTAATCGTTGGATGC",
-                   "TACCAGCTCTCCGAGTGGTGTGGATGTTTATTGGGCCTAAAGCATCCGTAGCTGG"
-                   "CTAGGTTAGTCCCCTGTTAAATCCACCGAATTAATCGTTGGATGC",
-                   "TACCCGCAGCTCAAGTGGTGGTCGCTATTATTGAGCCTAAAACGTCCGTAGTCGG"
-                   "CTTTGTAAATCCCTGGGTAAATCGGGTCGCTTAACGATCCGATTC"]
-        sample_ids = ["s1"]
-        table_exp = Table(data, otu_ids, sample_ids, sample_metadata=None)
-        table_obs = load_table(output_fp)
-        self.assertEqual(table_obs, table_exp)
+        biom_fp = launch_workflow(seqs_fp, output_fp, read_error, mean_error,
+                                  error_dist, indel_prob, indel_max,
+                                  trim_length, min_size, (ref_fp,),
+                                  ref_db_fp, negate, threads, delim)
+
+        # get the trimmed ground truth sequences
+        with open(origfilename, 'U') as f:
+            orig_seqs = [item[1] for item in parse_fasta(f)]
+        orig_seqs = [item[:trim_length].upper() for item in orig_seqs]
+
+        table_obs = load_table(biom_fp)
+        outseqs = table_obs.ids(axis='observation')
+        foundall = True
+        for cseq in orig_seqs:
+            if cseq.upper() not in outseqs:
+                foundall = False
+        # test we see all ground truth sequences
+        self.assertTrue(foundall)
+        # and don't see any additional sequences
+        self.assertTrue(len(outseqs) == len(orig_seqs))
```

Comment:
You can use `self.assertItemsEqual(outseqs, orig_seqs)` and no need to worry about the sort.